docs/developer-setup-examples.md

```diff
@@ -0,0 +1,107 @@
+# kubernetes and os setup examples
+
+This documents contains several examples to show working setups for a development environment
+
+## tested on ubuntu 18.04.4 with minikube 1.9.2 and zfs 0.7 and 0.8
+
+### Prerequisites
+* You have a zfs pool created on localhost, with pool name `zfspv-pool`; that is you have installed
+zfs-dkms (or zfs-fuse), zfs-zed and zfsutils-linux .deb packages. And then created the pool with `zpool create` command
+* You have followed the prerequisites stated in [Development Workflow](developer-setup.md)
+
+```
+wget https://github.com/kubernetes/minikube/releases/download/v1.9.2/minikube_1.9.2-0_amd64.deb
+sudo dpkg -i minikube_1.9.2-0_amd64.deb
+sudo minikube start --driver=none
+sudo chown -R $USER $HOME/.kube $HOME/.minikube
+
+kubectl apply -f https://raw.githubusercontent.com/openebs/zfs-localpv/master/deploy/zfs-operator.yaml
+kubectl get pods -n kube-system -l role=openebs-zfs
+
+export OPENEBS_NAMESPACE=openebs
+export KUBECONFIG=$HOME/.kube/config
+
+cd ~/go/src/github.com/openebs/zfs-localpv/tests
+
+ginkgo -v
```

Comment:
we can do `sudo -E env "PATH=$PATH" make ci` to run integration test after setting up the minikube and ZFS pool, see the doc : https://github.com/openebs/zfs-localpv/blob/master/docs/developer-setup.md#building-and-testing-your-changes.